testing/regress/ecl/despray.ecl

```diff
@@ -232,7 +232,7 @@ c8 := CATCH(NOFOLD(p8), ONFAIL(TRANSFORM(rec,
 
 
 // This should pass based on valid target file path and valid source address used
-DestFile9 := dropzonePath + 'test/' + prefix + File;
+DestFile9 := dropzonePath + 'test/' + WORKUNIT + '-' + File;
```

Comment:
minor: for consistency should use 'DestFileName' here instead WORKUNIT + '-' + File

NB: line 192, 214 should be changed too.

Reply:
Changed